backend/pretix/__init__.py

```diff
@@ -276,12 +282,18 @@ class CreateOrderTicketAnswer:
 @strawberry.input
 class CreateOrderTicket:
     ticket_id: str
-    attendee_name: str
+    attendee_name: AttendeeNameInput
     attendee_email: str
     variation: Optional[str] = None
     answers: Optional[List[CreateOrderTicketAnswer]] = None
     voucher: Optional[str] = None
 
+    def validate(self, errors: CreateOrderErrors) -> CreateOrderErrors:
+        if not self.attendee_name.validate():
```

Comment:
we should add the same check for emails

right now I believe we rely on pretix api to reject the email